installer.py

```diff
@@ -115,41 +98,34 @@ def chooseProduct():
     else:
         logos_error("Unknown product. Installation canceled!", "")
 
-    logging.info(f"Installing {config.FLPRODUCT} Bible Software")
-
+    # Set icon variables.
     if config.LOGOS_ICON_URL is None:
         config.LOGOS_ICON_URL = "https://raw.githubusercontent.com/ferion11/LogosLinuxInstaller/master/img/" + config.FLPRODUCTi + "-128-icon.png"
+    if config.LOGOS_ICON_FILENAME is None:
+        config.LOGOS_ICON_FILENAME = os.path.basename(config.LOGOS_ICON_URL)
 
 def getLogosReleaseVersion(releases):
-    logging.debug("Starting getLogosReleaseVersion()")
     TITLE=f"Choose {config.FLPRODUCT} {config.TARGETVERSION} Release"
     QUESTION_TEXT=f"Which version of {config.FLPRODUCT} {config.TARGETVERSION} do you want to install?"
     logos_release_version = curses_menu(releases, TITLE, QUESTION_TEXT)
 
+    logging.info(f"Release version: {logos_release_version}")
     if logos_release_version is not None:
         config.LOGOS_RELEASE_VERSION = logos_release_version
     else:
-        logging.critical("Failed to fetch LOGOS_RELEASE_VERSION.")
+        logos_error("Failed to fetch LOGOS_RELEASE_VERSION.")
```

Comment:
Is this right? Weren't we changing to logging.x?

Reply:
This of course could just be me misremembering the logging changes.

Reply:
logos_error now uses logging to record the main error message in the log file and display it on stderr. But I kept the function because it also gives the "get help" text and exits with an error code. I thought it seemed fitting for critical errors.

Reply:
Sounds good!